doc/sample_azure-keyvault.rst

```diff
@@ -22,7 +22,24 @@ for details on handling Azure Active Directory authentication with the Python SD
     credentials = UserPassCredentials(
         'user@domain.com',  # Your user
         'my_password',      # Your password
-        resource="https://vault.azure.net"
+        resource='https://vault.azure.net'
+    )
+
+    client = KeyVaultClient(
+        credentials
+    )
+
+.. code:: python
```

Comment:
I'm not sure this is needed, since I say "See above for details on creating different types of AAD credentials". Do you really think it's better to show both authentication here?